entrypoint.sh

```diff
@@ -30,6 +30,33 @@ CW_DIMENSIONS_DELIMITER="${CW_DIMENSIONS_DELIMITER:-" "}"
 CW_STATISTICS="${CW_STATISTICS:-"Average"}"
 CW_PERIOD="${CW_PERIOD:-360}"
 CW_POLL_PERIOD="${CW_POLL_PERIOD:-30}"
+LOG_LEVEL="${LOG_LEVEL:-"INFO"}" # OFF,ERROR,INFO,DEBUG
+
+# Validate LOG_LEVEL
+if [[ "${LOG_LEVEL}" != "OFF" && "${LOG_LEVEL}" != "ERROR" && "${LOG_LEVEL}" != "INFO" && "${LOG_LEVEL}" != "DEBUG" ]]; then
+    echo "LOG_LEVEL must be one of: OFF,ERROR,INFO,DEBUG"
+    exit 1
+fi
+
+# Logging functions, based on LOG_LEVEL environment variable
+log_error() {
+  if [[ "${LOG_LEVEL}" == "ERROR" || "${LOG_LEVEL}" == "INFO" || "${LOG_LEVEL}" == "DEBUG" ]]; then
+    echo -e "\033[1;31m$(date -u -I'seconds') ERROR:\033[0m ${1}"
+  fi
+}
+
+log_info() {
+  if [[ "${LOG_LEVEL}" == "INFO" || "${LOG_LEVEL}" == "DEBUG" ]]; then
+    echo -e "\033[1;32m$(date -u -I'seconds') INFO:\033[0m ${1}"
+  fi
+}
+
+log_debug() {
+  # Maintain backwards compat for VERBOSE flag
+  if [[ "${LOG_LEVEL}" == "DEBUG" || "${VERBOSE}" == true ]]; then
+    echo -e "\033[1;33m$(date -u -I'seconds') DEBUG:\033[0m ${1}"
```

Comment:
I usually associate yellow with warnings. Maybe we should keep debug as white?

Reply:
Will do.

Reply:
Done